packs/tesla/pack.yaml

```diff
@@ -3,6 +3,6 @@ name : tesla
 description : car horn automation
 keywords:
   - tesla
-version : 0.1
+version : 0.1.1
```

Comment:
Since this is a major change to the functionality I would prefer 0.2.0 as the version number if we're following semver